src/cf.data.pre

```diff
@@ -4995,10 +4995,33 @@ DOC_START
 		[http::]<pt	Peer response time in milliseconds. The timer starts
 				when the last request byte is sent to the next hop
 				and stops when the last response byte is received.
-		[http::]<tt	Total time in milliseconds. The timer
-				starts with the first connect request (or write I/O)
-				sent to the first selected peer. The timer stops
-				with the last I/O with the last peer.
+		[http::]<tt	Total peering time in milliseconds.
```

Comment:
If that were true, you would not have insisted on me writing "server or peer" in all previous PRs.

Reply:
> > Alex: I have changed that brief `<tt` description to use the terminology you prefer: "Total time spent forwarding to origin servers or cache_peers (milliseconds)." I hope this addresses your concern.

> Amos: If that were true, you would not have insisted on me writing "server or peer" in all previous PRs.

I do not know what "it" refers to, but can we focus on moving forward this stalled PR rather than on my alleged past actions in other PRs, please? AFAICT, this change request has been addressed two months ago, using phrasing nearly identical to the one you have suggested. Please resolve this specific change request and, if possible, dismiss your negative PR review.